docs/docs/deploy/railway.md

```diff
@@ -0,0 +1,79 @@
+---
+sidebar_position: 5
+title: 🛤️ Railway
+---
+
+# Railway 🛤️
+
+[Railway](https://railway.app) is a cloud platform that can build, deploy, and monitor your HTTP applications (like APIs made with Dart Frog) with minimal setup. It embraces private networks so you can link databases with other services easily and securely without needing much knowledge of network security. It also features:
+
+- Secret management
+- Autoscaling (both vertically and horizontally), as well as sleeping services when not in use.
+- Auto deployment from GitHub
+- Usage based pricing
+- Hundreds of templates for quickly deploying ready-to-use software
+- Health checks, instant rollbacks, environments, and more
+
+This guide will show you how to auto-build-and-deploy to Railway using it's integrated build system. However, Railway is also able to deploy pre-built Docker images. If you rather, you could build your application in an external system (like GitHub Actions) and [deploy to Railway with that image](https://docs.railway.app/overview/the-basics#services).
```

Comment:
The `If you rather, you could build your application...` reads weirdly, or is that just me?

Reply:
```suggestion
This guide will show you how to auto-build-and-deploy to Railway using it's integrated build system. However, Railway is also able to deploy pre-built Docker images. You could also build your application in an external system (like GitHub Actions) and [deploy to Railway with that image](https://docs.railway.app/overview/the-basics#services).
```